### PR TITLE
Created folder markup-float,add file index.html made with float

### PR DESCRIPTION
**Pull request by AndriiChuprina**

### CODE REVIEW

File: markup-float/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+
+      .container {
```
Comment (dmitry-darly-solutions) [inline markup-float/index.html]: move styles to css file

File: markup-float/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  </style>
+</head>
+<body>
+    <div class="container">
+         <div class="width33" style="background-color:rgb(250,235,216)"></div>
+         <div class="width33" style="background-color:rgb(245,245,221)"></div>
+         <div class="width33" style="background-color:rgb(254,227,198)"></div>
+    </div>
+
```
Comment (dmitry-darly-solutions) [inline markup-float/index.html]: remove all empty lines in html

Comment (AndriiChuprina) [reply markup-float/index.html]: done

File: markup-float/index.html (hunk 1 above)
Comment (AndriiChuprina) [reply markup-float/index.html]: done

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**